src/test/java/io/airlift/compress/lzo/TestLzopCodec.java

```diff
@@ -68,4 +75,34 @@ protected Decompressor getVerifyDecompressor()
     {
         return new HadoopCodecDecompressor(verifyCodec);
     }
+
+    @Test
+    public void testDecompressNewerVersion()
+            throws IOException
+    {
+        // lzop --no-checksum -o test-no-checksum.lzo test
```

Comment:
Would be good to record `lzop` version used to produce the files.

Reply:
It's actually recorded inside the files.

Reply:
I know. It might be however that 2 different impls record same version. Or that I don't know where to obtain particular version from.